va/validation-authority_test.go

```diff
@@ -627,7 +612,7 @@ func TestDNSValidationBadDNSSEC(t *testing.T) {
 
 func TestDNSValidationNoServer(t *testing.T) {
```

Comment:
I'm conflicted about whether this test is appropriate now that it's a mock.
Thoughts?